packages/office-addin-dev-certs/src/install.ts

```diff
@@ -13,6 +13,8 @@ function getInstallCommand(caCertificatePath: string, machine: boolean = false):
          return `powershell -ExecutionPolicy Bypass scripts\\install.ps1 ${machine ? "LocalMachine" : "CurrentUser"} '${caCertificatePath}'`;
       case "darwin": // macOS
          return `sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain '${caCertificatePath}'`;
+      case "linux":
+         return `sudo mv ${caCertificatePath} /usr/local/share/ca-certificates && sudo /usr/sbin/update-ca-certificates`;
```

Comment:
This should be copy instead of a move since other packages try to read from the original location

Reply:
Done in https://github.com/OfficeDev/Office-Addin-Scripts/pull/156/commits/a7747c876cbcf7a6c65013824cf10556aa5c9079